Makefile.vars

```diff
@@ -1,3 +1,4 @@
 QEMU_VERSION := 4.2.0
 LINUX_VERSION := 5.4.30
 BUSYBOX_VERSION := 1.31.1
+PULSEAUDIO_VERSION := 13.99.1
```

Comment:
You also need to add pulseaudio to `URLS` in the `Makefile`. See other files there - url should point at a signature file (if exists, which doesn't look to be the case) or alternatively add a `.sha512` file with a hash.

BTW this should be detected by travis, as the build would fail now. But for some reason travis didn't built it at all. Looking at a log in their web interface I see your build marked as "Abuse detected"!?